task2.Rmd

```diff
@@ -0,0 +1,137 @@
+---
+title: "task2"
+author: "Jan Prugarewicz"
+date: "25 11 2020"
+output: html_document
+---
+
+# Zadanie 2
+
+## Treść zadania
+
+Na podstawie notowań pewnej spółki we wrześnkiu 2020 (Tauron Polska Energia Spółka Akcyjna):
++ zilustruj jak liczba transakcji na danym instrumencie rozkłada się w czasie w ramach giełdowej sesji (chodzi zarówno o podział między 3 fazami: fixingiem otwarcia, notowaniami ciągłymi, fixingiem zamknięcia, jak i rozkład liczby transakcji w czasie fazy notowań ciągłych),
++ wybierz dzień i 2-godzinny przedział czasu, w ramach którego
+  - wyznacz liczby transakcji zawieranych w kolejnych 1-minutowych odcinkach czasu (120 liczb),
+  - zamodeluj ww. liczby za pomocą rozkładu Poissona,
+  - porównaj rozkład modelowy z danymi.
+  
+## Rozwiązanie podpunktu a
+```{r setup zadanie2}
```

Comment:
setupu bym nie pokazywała w raporcie, tam była na to flaga. Stosowałam ją na pewno ;)

Reply:
w ogóle na raporcie bym pokazywała tylko wyniki, o które proszą w treści zadania. kod, który je generuje nie